docs/deployment.md

```diff
@@ -30,10 +30,10 @@ maintainer decides it's time to release.
 Make sure it's installed and all dependencies are met. Consult
 `scripts/Jenkinsfile.deploy` and set up all the build parameters on your server.
 
-The deployment is performed over SSH, so make sure that CI have SSH access to
-the target machine as the same `cor-site` user and a permission to execute
-`sudo /sbin/stop cor-site` and `sudo /sbin/start cor-site` commands (and nothing
-else).
+The deployment is performed over SSH, so make sure that CI server have SSH
+access to the target machine as the same `cor-site` user, and that `cor-site`
+have a permission to execute the commands `sudo /sbin/stop cor-site` and
```

Comment:
`cor-site` is a "user", so he "has" permission. I also don't think the article is needed here. So it all should read "…and `cor-site` has permission to execute…".